app/models/participant_profile/npq.rb

```diff
@@ -3,6 +3,7 @@
 class ParticipantProfile::NPQ < ParticipantProfile
   self.ignored_columns = %i[mentor_profile_id school_cohort_id]
   belongs_to :school, optional: true
+  belongs_to :npq_course, optional: true
```

Comment:
Will we eventually make it non-optional here?

Reply:
yea i guess we should change that when we get there